lib/auth-user.js

```diff
@@ -1,7 +1,18 @@
+const getUserDetails = require('./get-user-details');
+
 module.exports = function(req, res, next) {
-  // TODO: user req.query.access_token to validate the user againts the osm api
   // TODO: think about a way to support robot users that do not have an OSM account
   // TODO: think about different permissions types based on the user
-  req.user = 'userone';
-  next();
+
+  // These two tokens are stored in localStorage after signing in with osm-auth
+  const oauthToken = req.query.oauth_token;
```

Comment:
@batpad doubt,
Should we be checking if oauth_token is supplied, and if not handle it?

Reply:
👍 on it